### DISCUSSION TITLE
IPv6 EUI-64 Modeling

### DISCUSSION THREAD

**Discussion by DCAuto** [netbox-community/netbox#15711] [Q&A]
We're currently testing an IPv6 EUI-64 deployment within one of our data-centers and I was wondering if anybody has had a similar deployment and how they tracked it in netbox.

The setup is as follows:
 - Each interface of a router is assigned a separate /64 IPv6 prefix
 - At the other end of the interface is a hypervisor with many vms
 - Each vm gets an IPv6 address automatically assigned based on the mac-address (per EUI-64 standards)

The issue we are running in to is I can't find a good way to map the /64 to the physical interface. I have tried mapping the ".0" address but I get a "can't assign network address" error even when the prefix is set to a "pool". I have read several recent posts discussing this error and it looks like that functionality isn't going to change. *

I suppose I could use the ".1" address but in the configuration it references "fd00:1234:1234:1234::/64" and not "fd00:1234:1234:1234::1/64"

Any thoughts on how to solve this issue?

**Comment by markkuleinio** [comment]
In other words, you have a routed /64 network connecting the router interface and the hypervisor host, and similarly for each hypervisor separately.

Thus, the router has one address from the /64 (I'd use ::1), and the rest of the addresses are autoconfigured by the VMs.

In NetBox you have (for one hypervisor)
- a prefix fd00:1234:1234:1234::/64
- IP address fd00:1234:1234:1234::1/64, assigned in the router interface
- VM IP addresses similarly, if you'll have them in NetBox

fd00:1234:1234:1234::0/64 is also the subnet router anycast address and is not assignable on a host interface.
